src/ai/backend/manager/models/etcd.py

```diff
@@ -56,6 +56,13 @@ class ContainerRegistry(graphene.ObjectType):
     hostname = graphene.String()
     config = graphene.Field(ContainerRegistryConfig)
 
+    class Meta:
+        interfaces = (graphene.relay.Node,)
+
+    # @classmethod
+    # def get_node(cls, info: graphene.ResolveInfo, id):
+    #     raise NotImplementedError
+
```

Comment:
Is this code for graphql node?

Reply:
`graphene` requires relay nodes to have `get_node()` class method to be implemented. This method is used to query the object directly by unique id. But yet we do not support this feature, which means it is not necessary.

Please refer: https://docs.graphene-python.org/en/latest/relay/nodes/#nodes